packages/core/admin/admin/src/components/GuidedTour/init.js

```diff
@@ -3,13 +3,19 @@ import persistStateToLocaleStorage, {
   COMPLETED_STEPS,
   CURRENT_STEP,
   SKIPPED,
+  VISIBLE,
 } from './utils/persistStateToLocaleStorage';
 
 const init = initialState => {
   const copyInitialState = { ...initialState };
   const guidedTourLocaleStorage = persistStateToLocaleStorage.get(COMPLETED_STEPS);
   const currentStepLocaleStorage = persistStateToLocaleStorage.get(CURRENT_STEP);
   const skippedLocaleStorage = persistStateToLocaleStorage.get(SKIPPED);
+  const isVisibleLocaleStorage = persistStateToLocaleStorage.get(VISIBLE);
```

Comment:
I have noticed a bug where `COMPLETED_STEPS` returns true sometime can you check where the issue is coming from?

Reply:
I was unable to reproduce this, I asked @vincentbpro to try also!